spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/configuration/TaskServiceDependencies.java

```diff
@@ -216,20 +217,25 @@ public TaskExecutionCreationService taskExecutionRepositoryService(TaskRepositor
 		return new DefaultTaskExecutionRepositoryService(taskRepository);
 	}
 
+	@Bean
+	TaskAppDeploymentRequestCreator taskAppDeploymentRequestCreator(
```

Comment:
Is there any reason why this is not `public`?

Reply:
oops updated!